src/views/Records/Record.vue

```diff
@@ -190,6 +201,31 @@
                 }
               })
             },
+            async requestOwnership() {
+              let _module = this;
+              const recordID =  _module.currentRecord['fairsharingRecord'].id;
+              const claim = await client.claimRecord(recordID, _module.user().credentials.token);
+              if (claim.error) {
+                _module.error = "Sorry, your request to claim this record failed. Please contact us.";
+                _module.canClaim = false;
+              } else {
```

Comment:
can you please put else's on new lines ? :D
The official name is Stroustrup styling.
Here in case you are interested: https://eslint.org/docs/rules/brace-style